tests/setup-debuggertests.cmd

```diff
@@ -0,0 +1,90 @@
+@if not defined __echo @echo off
```

Comment:
Not at all required for this checkin, but a suggestion if you are iterating on this... could these scripts be replaced by a small C# app that is identical across platforms? Presumably we need a working runtime to run xunit and mdbg, so we've already crossed the bridge that all debugger tests require at least a semi-functional runtime to be available (although it doesn't have to be the runtime that is being tested).

Doing these tasks with scripts means that we have to write it once for each different shell, and in some cases there are other somewhat arbitrary platform differences such as downloading with curl vs. inline powershell., and unpacking with tar vs. zip. The more consistent we can be the better IMO, unless the implementation in C# winds up being more complicated then aggregate of all the scripts.